src/components/YextEntityFieldSelector.tsx

```diff
@@ -42,41 +43,51 @@ export const YextEntityFieldSelector = <T extends Record<string, any>>(
     label: props.label,
     render: ({ field, value, onChange }: RenderProps) => {
       const filteredEntityFields = getFilteredEntityFields(props.filter);
+      const [entityFieldMode, setEntityFieldMode] = useState<boolean>(true);
+
+      const toggleEntityFieldMode = () => {
+        setEntityFieldMode((prevMode) => !prevMode);
+        onChange({
```

Comment:
This makes it so when entityFieldMode is toggled off or on, both fields reset to ""?
I know we discussed this with Aaron a lot but I don't remember where we ended up. My concern with this is that an entity that doesn't have data for this field will return ""

Reply:
Oh hmm let me look at slack threads + whats currently returned w/ nothing being manually selected by user

Reply:
https://github.com/user-attachments/assets/a615910a-614c-4382-b9a8-9122541c8fec